ros2_ws/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
While not strictly an error, it's good practice to specify the project language explicitly when using `cmake_minimum_required`.  Add `CXX` to indicate that the project is written in C++:  `cmake_minimum_required(VERSION 3.5 CXX)`